packages/paste-core/components/visual-picker/stories/checkbox.stories.tsx

```diff
@@ -0,0 +1,522 @@
+// eslint-disable-next-line eslint-comments/disable-enable-pair
+/* eslint-disable import/no-extraneous-dependencies */
+import * as React from 'react';
+import {Anchor} from '@twilio-paste/anchor';
+import {Text} from '@twilio-paste/text';
+import {Avatar, AvatarGroup} from '@twilio-paste/avatar';
+import {MediaObject, MediaBody, MediaFigure} from '@twilio-paste/media-object';
+import {AcceptIcon} from '@twilio-paste/icons/esm/AcceptIcon';
+import {UserIcon} from '@twilio-paste/icons/esm/UserIcon';
+import {Box} from '@twilio-paste/box';
+
+import {VisualPickerCheckboxGroup, VisualPickerCheckbox} from '../src';
+
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Visual Picker/Checkbox',
+  component: VisualPickerCheckboxGroup,
+};
+
+export const VerticalCheckboxGroup = (): React.ReactNode => {
+  const [checked1, setChecked1] = React.useState(true);
+  const [checked2, setChecked2] = React.useState(false);
+  const [checked3, setChecked3] = React.useState(false);
+  const handleChange1 = (event: React.ChangeEvent<HTMLInputElement>): void => setChecked1(event.target.checked);
+  const handleChange2 = (event: React.ChangeEvent<HTMLInputElement>): void => setChecked2(event.target.checked);
+  const handleChange3 = (event: React.ChangeEvent<HTMLInputElement>): void => setChecked3(event.target.checked);
+  return (
+    <VisualPickerCheckboxGroup
+      orientation="vertical"
+      legend={
+        <Text as="span" color="currentColor">
+          This is a Visual Picker Checkbox Group label with a <Anchor href="http://paste.twilio.com">link</Anchor>
+        </Text>
+      }
+      name="visual-picker"
+      helpText="Help text should go here."
+    >
+      <VisualPickerCheckbox labelText="SQL" checked={checked1} onChange={handleChange1}>
+        <MediaObject verticalAlign="center">
+          <MediaFigure spacing="space50">
+            <Avatar size="sizeIcon90" name="B T" color="decorative20" />
+          </MediaFigure>
+          <MediaBody>
+            <Text as="div" fontWeight="fontWeightSemibold">
+              SQL
+            </Text>
+            <Text as="div">Create any trait from your data warehouse by writing SQL.</Text>
```

Comment:
suggestion: Set the secondary text in all stories and examples to colorTextWeak. We talked about changing it in today's design crit. https://www.figma.com/file/T8blAjMrVlBc0sPX92T2PQ/(feat)Visual-Picker---add-component?type=design&node-id=32302-162413&mode=design&t=WlZm7oXEYhSv9BnW-4